build.sbt

```diff
@@ -19,13 +19,15 @@ resolvers += Resolver.sonatypeRepo("releases")
 resolvers += Resolver.sonatypeRepo("snapshots")
 
 libraryDependencies ++= Seq(
-  "org.virtuslab" %% "unicorn" % "1.3.0",
-  "org.scalatestplus.play" %% "scalatestplus-play" % "3.0.0" % Test,
-  "com.typesafe.play" %% "play-test" % "2.6.20" % "test",
-  "com.typesafe.play" %% "play-json-joda" % "2.6.10",
-  "com.h2database" % "h2" % "1.4.187" % "test"
+  "org.virtuslab" %% "unicorn" % "1.3.3-SNAPSHOT",
```

Comment:
remove SNAPSHOT

Reply:
for now it could be snapshot until we release it